.circleci/config.yml

```diff
@@ -88,6 +73,11 @@ jobs:
               /tmp/cloudify-manager-install/packaging/fetch_requirements --edition premium -b ${CIRCLE_BRANCH} >~/fetch_requirements.log
               cat ~/fetch_requirements.log
             popd
+      - run: sudo yum install rpmdevtools -y
+      - run: sudo chmod a+wx /opt
```

Comment:
same as everywhere else, we make the venv on the base system's `/opt`, (which is the same path that they will be used from) to avoid having to make the venvs relocatable